### PR TITLE
Added support for H2 tag perms

### PR DESCRIPTION
**Pull request by maxsam4**

### CODE REVIEW

File: utils/permissions/src/main.rs [hunk 1]
```diff
@@ -109,21 +110,23 @@ fn parse_permissions(doc: EnumDoc) -> Vec<(String, String)> {
...
                 })
-                // Look for a ul in the contents before the next h1
+                // Look for a ul in the contents before the next h1/h2
                 .for_each(|(i, _)| {
                     children[i + 1..]
                         .iter()
                         // Stop once we hit another h1
-                        .take_while(|element| !h1_selector.matches(element))
+                        .take_while(|element| {
+                            !(h1_selector.matches(element) || h2_selector.matches(element))
```
Comment (Centril) [inline utils/permissions/src/main.rs:128]: maybe extract the inner parens to a closure and use in the filter above as well?

Comment (maxsam4) [reply utils/permissions/src/main.rs:128]: Tried that earlier but compiler started complaining about lifetimes and I went: not worth it :)

**Review by Centril — APPROVED** [review]

File: utils/permissions/src/main.rs (hunk 1 above)
Comment (Centril) [reply utils/permissions/src/main.rs]: Seems like it worked out in the end, neat :)

Comment (maxsam4) [reply utils/permissions/src/main.rs]: Yeah but I had to deference the element which I believe will cause a shadow copy to be created so not ideal but whatever lol

**Review by adamdossa — APPROVED** [review]

**Pull request merged**